bin/oneoff/nces_data/import_school_stats_by_years_2017_2018_ccd

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+
+SURVEY_YEAR = '2017-2018'.freeze
+
+GRADE_MAP = {
+  'Prekindergarten' => 'PK',
+  'Kindergarten' => 'KG',
+  '1st Grade' => '01',
+  '2nd Grade' => '02',
+  '3rd Grade' => '03',
+  '4th Grade' => '04',
+  '5th Grade' => '05',
+  '6th Grade' => '06',
+  '7th Grade' => '07',
+  '8th Grade' => '08',
+  '9th Grade' => '09',
+  '10th Grade' => '10',
+  '11th Grade' => '11',
+  '12th Grade' => '12',
+  '13th Grade' => '13'
+}
+
+VIRTUAL_SCHOOL_MAP = {
+  'A virtual school' => 'Yes',
+  'Not a virtual school' => 'No'
+}
+
+TITLE_I_MAP = {
+  '1-Eligible for Title I TAS provides no program' => '1',
+  '2-Eligible for Title I TAS provides TAS program' => '2',
+  '3-Eligible for Title I SWP provides TAS program' => '3',
+  '4-Eligible for Title I SWP provides no program' => '4',
+  '5-Eligible for Title I SWP provides SWP program' => '5',
+  '6-Not eligible for either TAS or SWP' => '6'
+}
+
+COMMUNITY_TYPE_MAP = {
+  '11-City: Large' => 'city_large',
+  '12-City: Mid-size' => 'city_midsize',
+  '13-City: Small' => 'city_small',
+  '21-Suburb: Large' => 'suburban_large',
+  '22-Suburb: Mid-size' => 'suburban_midsize',
+  '23-Suburb: Small' => 'suburban_small',
+  '31-Town: Fringe' => 'town_fringe',
+  '32-Town: Distant' => 'town_distant',
+  '33-Town: Remote' => 'town_remote',
+  '41-Rural: Fringe' => 'rural_fringe',
+  '42-Rural: Distant' => 'rural_distant',
+  '43-Rural: Remote' => 'rural_remote'
+}.freeze
+
+# @param unsanitized [String, nil] the unsanitized string
+# @returns [String, nil] the sanitized version of the string, with newlines stripped and double
+#   quotations escaped. Returns nil on nil input
+def sanitize_string_for_db(unsanitized)
+  return nil if unsanitized.nil?
+  unsanitized.tr('="', '')
+end
+
+# Context: used to be in "wide" format, switched to long.
+# Used table generator on website to get data I want.
```

Comment:
It might be a good idea to include which website you used in this documentation, in case someone wants to use this script again and needs access to it.

Reply:
Good idea, also will add.